database/migrations/1548428537_create_cluster_status_history_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `cluster_status_history` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `cluster_id` int(10) unsigned NOT NULL,
```

Comment:
Do we want to keep cluster status history record beyond the lifetime of the cluster record? If not let's add a foreign key to `cluster_id`

Reply:
I deliberately made the decision not to add foreign keys:

1. With the deleted at behavior clusters never get deleted from the clusters table
2. The purpose of this table is to give an overview of the cluster status changes (eg. ERROR->DELETING). If the cluster ever gets deleted, this information is lost.
3. This is a temporary solution, so I'm not afraid of the table getting large.

Please confirm if the purpose of this solution changed @stoader @ahma